pkg/apiserver/apiserver.go

```diff
@@ -96,6 +96,11 @@ type APIGroupVersion struct {
 	Context api.RequestContextMapper
 
 	MinRequestTimeout time.Duration
+
+	// SubresourceOverrides contains the GroupVersionKind overrides for each subresource that is
+	// accessible from this API group version. The key of this map should be the path of the
+	// subresource. The keys here should match the keys in the Storage map above for subresources.
+	SubresourceGroupVersionKind map[string]unversioned.GroupVersionKind
```

Comment:
Why did you need to add it both here and in genericapaiserver? Should only need to be added in one place-- genericapiserver is preferable.